android/app/src/main/res/values-ar/strings.xml

```diff
@@ -856,6 +856,7 @@
 	<string name="type.amenity.parking">موقف سيارات</string>
 	<string name="type.amenity.parking.fee">موقف سيارات</string>
 	<string name="type.amenity.parking.multi.storey">موقف سيارات متعدد الطوابق</string>
+	<string name="type.amenity.parking.multi.storey.fee">موقف سيارات متعدد الطوابق</string>
```

Comment:
Maybe create an issue to automatically fall back to the base type translation in such cases? It also saves a bit of space by avoiding duplicate strings.

Reply:
Well, we had the idea before. Should _override_ standard Android/iOS mechanics.